build/Microsoft.DotNet.Cli.Monikers.props

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <SdkBrandName>Microsoft .NET Core 1.0.3 - SDK RC 4</SdkBrandName>
```

Comment:
That would make for an unruly string. Will need to discuss the idea with others.